Fix transmission -> webrtc (#25230)
mdn/content: files/en-us/web/api/mediacapabilities/encodinginfo/index.md

        - `record`
          - : Represents a configuration for recording of media, e.g. using {{domxref("MediaRecorder")}}.
        - `webrtc`

>           - : Represents a configuration meant to be transmitted over electronic means (e.g. using {{domxref("RTCPeerConnection")}}). **Note:** doesn't work on Firefox, uses `transmission` instad.

Author: ```suggestion
          - : Represents a configuration meant to be transmitted over electronic means (e.g. using {{domxref("RTCPeerConnection")}}). **Note:** doesn't work on Firefox, uses `transmission` instead.
```

Reviewer: instead

>           - : Represents a configuration meant to be transmitted over electronic means (e.g. using {{domxref("RTCPeerConnection")}}). **Note:** doesn't work on Firefox, uses `transmission` instead.

Reviewer 2: ```suggestion
          - : Represents a configuration meant to be transmitted over electronic means (e.g. using {{domxref("RTCPeerConnection")}}). **Note:** Firefox uses `transmission` for this type, and `webrtc` does not work.
```
